src/pybind/mgr/dashboard/frontend/e2e/page-helper.po.ts

```diff
@@ -64,7 +64,7 @@ export abstract class PageHelper {
         if (tableCount > 0) {
           const progressBars = element.all(by.css('cd-table datatable-progress'));
           await progressBars.each(async (progressBar) => {
-            await browser.wait(EC.not(EC.presenceOf(progressBar)), Helper.TIMEOUT);
+            await this.waitStalenessOf(progressBar);
```

Comment:
You'll get a problem here for two reasons:

1. `waitStalenessOf` doesn't seem to exist anymore
2. `this` cannot be accessed here, as it requires an instance and the function it is used in (L50) is static.

You'll either need to make `waitStaleness` static or just not optimize this part of the code.